src/sokoban.cpp

```diff
@@ -1,16 +1,19 @@
 #include "sokoban.hpp"
 
-Sokoban::Sokoban(std::vector<std::vector<std::string>> levels) {
+Sokoban::Sokoban(std::vector<std::vector<std::string>> levels) 
+{
```

Comment:
Please follow the [style guide](https://github.com/ggorlen/cs195-project/blob/main/documentation/CODE_STYLE.md#c) -- we're using Egyptian rather than Allman braces and 4-spaces rather than tabs.

A PR that reformats all of the existing code creates too large a footprint -- this isn't related to the feature, and if we do agree on a total reformat, that should be its own PR.